src/randoop/sequence/AbstractGenerator.java

```diff
@@ -113,40 +117,49 @@
   public static Sequence currSeq = null;
   
   /**
-   * The list of final sequences that are printed out as JUnit tests (i.e. Randoop's output). 
+   * The list of failure sequences to be saved as JUnit tests
+   */
+  public List<ExecutableSequence> outErrorSeqs = new ArrayList<>();
+
+  /**
+   * The list of regression sequences to be save as JUnit tests
```

Comment:
"save" => "saved".  Also, here and above I would find "to be output" or "to be written" clearer than "to be saved" since "save" also has different meanings in computing, such as saving in a data structure.